### ISSUE TITLE
USER STORY: Site Navigation

### ISSUE THREAD

**Issue by DarrachBarneveld** [DarrachBarneveld/Retroverse-Odyssey#1]
As a **User**, I can intuitively navigate around the site so that I can find content.

**Acceptance Criteria:**
1. A navbar should be visible on all pages allowing the user to navigate to each section of the website.

**Tasks**
- [x] Create a Logo
- [x] Link the relevant pages to the menu buttons. Page1 - Play. Page2 - Rules. Page3 - Controls. Page4 - About us. Page5 - My trophies.

**Comment by ysgurjar** [comment]
Hi, 

I slightly disagree that 80s game came with Navbars. They usually presented limited options  "in your face". Something similar to this one.

![Image](https://github.com/DarrachBarneveld/Retroverse-Odyssey/assets/94364828/95b18a0c-91e2-4d83-9433-80d007fc787c)

@DebbieBergstrom  : Which part of the site navigation are you working on ?

**Comment by DebbieBergstrom** [comment]
I agree =) I think your wireframes and the way you thought about it is great =)
